Hatnya/Screens/CreateStudy/UIComponent/CyclePickerView.swift

```diff
@@ -0,0 +1,105 @@
+//
+//  CyclePickerView.swift
+//  Hatnya
+//
+//  Created by kelly on 2022/07/20.
+//
+
+import UIKit
+
+class CyclePickerView: UIView {
+    var isShown = false
+    let cycles = ["1", "2", "3", "4"]
+    
+    private let titleLabel: UILabel = {
+        let label = UILabel()
+        label.text = "반복"
+        label.font = UIFont.systemFont(ofSize: 17)
+        return label
+    }()
+    
+    var selectedCycleLabel: UILabel = {
+        let label = UILabel()
+        label.text = "1주"
+        label.font = UIFont.systemFont(ofSize: 17)
+        label.textColor = .systemBlue
+        return label
+    }()
+    
+    let pickerView: UIPickerView = {
+        let picker = UIPickerView()
+        return picker
+    }()
+    
+    private let hStackView: UIStackView = {
+        let stackView = UIStackView()
+        stackView.axis = .horizontal
+        stackView.alignment = .fill
+        stackView.distribution = .equalSpacing
+        stackView.spacing = 1
+        return stackView
+    }()
+    
+    private let vStackView: UIStackView = {
+        let stackView = UIStackView()
+        stackView.axis = .vertical
+        stackView.alignment = .fill
+        stackView.distribution = .equalSpacing
+        stackView.spacing = 10
+        return stackView
+    }()
+    
+    override init(frame: CGRect) {
+        super.init(frame: frame)
+        
+        render()
+    }
+    
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
```

Comment:
p3;

오잉 여기서 LintError를 피하셨나요 ??
전 못피해서 ...
```suggestion
    required init?(coder: NSCoder) {
        super.init(coder: coder)
    }
```
아마 요렇게 하면 될겁니다 !

Reply:
흠 LintError가 혹시 노란 줄로 나타나는 거라면 발생하지 않은거 같은데..!

<img width="254" alt="스크린샷 2022-07-22 오전 10 46 04" src="https://user-images.githubusercontent.com/80809782/180343852-d4b6936f-4856-432a-88de-54218763a9df.png">
<img width="610" alt="스크린샷 2022-07-22 오전 10 45 10" src="https://user-images.githubusercontent.com/80809782/180343745-5de3c6ae-a4f5-49b3-8332-a725e8a78e8e.png">

그래도 수정하는게 좋을까요? 아니면 혹시 제가 린트에러를 발견 못하고 있는걸까요?

Reply:
전 빨간줄로 나는 에러였어요 ..!
fatalError가 앱을 강제로 종료시키는 함수라 린트에서 막는 것 같더라구요 !

Reply:
p2;

해당 부분은 fatalError 대신 내용을 채워넣길 권장하는 거라 수정하시면 좋을 것 같습니다!

Reply:
[63b5c34](https://github.com/DeveloperAcademy-POSTECH/MC3-Team1-Hatnya/pull/25/commits/63b5c34dd3049cad54db3e14d511adacb43919dc)
수정 완료했습니다!!